toonz/sources/stdfx/igs_resource_msg_from_err_unix.cpp

```diff
@@ -1,5 +1,7 @@
+#ifndef FREEBSD
 #define _POSIX_C_SOURCE 200809L /* int strerror_r() */
 #undef _GNU_SOURCE
+#endif
```

Comment:
Can you check if this is enough? I can check if it's enough for linux as well, in a bit.

```suggestion
#define _XOPEN_SOURCE 700 /* int strerror_r() */
#undef _GNU_SOURCE
```

Reply:
I prefer to not play with these defines at all.
Probably better idea to localize this workaround by using ```#ifdef "musl libc"``` equivalent code.

Reply:
No, it isn't. Please see https://man.voidlinux.org/feature_test_macros.7

glibc always provides their incompatible definition, unless `_GNU_SOURCE` is undefined, so this is completely necessary. Defining `_XOPEN_SOURCE 700` afterwards is just good practice, and should guarantee the `isascii()` definition for FreeBSD libc.

Reply:
I think problem is in ```_POSIX_C_SOURCE```, ```_GNU_SOURCE``` is "safe" to define/undefine, at least on FreeBSD.
Can you check build without ```_POSIX_C_SOURCE``` on system with musl libc?

Reply:
Defining `_XOPEN_SOURCE` gives you basically what `_POSIX_C_SOURCE` does, plus some extras. The issue is that defining `_POSIX_C_SOURCE` instead of `_XOPEN_SOURCE` means some specific symbols, like `isascii()`, might not be defined, for strict POSIX compliance.

The correct solution here is to use `_XOPEN_SOURCE`. It is required.